BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -3,6 +3,24 @@ import UIKit
 
 public struct AccessibilityElement: Element {
 
+    /// Used to provide addtional functionality to assistive technogies beyond your accessible UI.
+    public struct CustomAction {
+        public typealias OnActivation = () -> Bool
```

Comment:
Not sure on all the use cases, but should we input a `CustomAction` here to mirror `UIAccessibilityCustomAction.Handler`?

Reply:
It would more closely mirror apple's implementation, but all it would gain you is the name and the callback itself. You'll have the name when creating the action and I can't see why having access to the callback within itself would be useful unless you wanna do some weird recursive something that would probably be unwise anyway,

Reply:
I'm good with omitting. We can always add later if there is a good use case.